docs_src/OVMS/pipelinerun.md

```diff
@@ -74,64 +72,89 @@ The following are the optional parameters that you can provide as input to `dock
 - `--color-width`, `color-height`, and `color-framerate`: Allows you to customize the settings of the color frame output from the Intel® RealSense™ Cameras. This parameter will overwrite the default value of RealSense gstreamer. Use `rs-enumerate-devices` to look up the camera's color capability.
 
 
+### Supporting different programming languages for OVMS grpc client
+We are supporting multiple programming languages for OVMS grpc client. Currently we are supporting grpc-python and grpc-go. The scripts to start pipelines above would start grpc-python as default. [See more on supporting different language](./supportingDifferentLanguage.md)
+
+
+### Supporting different models for OVMS grpc python client
+With OVMS grpc-python client, you can configure to use different model to run the inferencing pipeline. The scripts to start pipelines above would start grpc-python using `instance-segmentation-security-1040` model as default. [See more on supporting different model](./supportingDifferentModel.md)
+
+
 ### Status of Running a Pipeline
     
 When you run the pipeline, the containers will run.
     
 Check if the pipeline run is successful: 
 
 ```bash
-docker ps --format 'table{{.Image}}\t{{.Status}}\t{{.Names}}' -a
+docker ps -a
```

Comment:
The previous command has a simpler output for users. Is there a reason we want to change this?

Reply:
yes, we show user how to see container log by using containerID, previous format doesn't show containerID

Reply:
updated to use containerName for query docker logs file